neard/src/lib.rs

```diff
@@ -193,6 +196,10 @@ pub fn apply_store_migrations(path: &String, near_config: &NearConfig) {
         // version 18 => rectify inflation: add `timestamp` to `BlockInfo`
         migrate_18_to_rectify_inflation(&path);
     }
+    #[cfg(feature = "protocol_feature_block_header_v3")]
+    if db_version <= 18 {
+        migrate_18_to_new_validator_stake(&path);
+    }
```

Comment:
this should be done in the nightly_protocol block

Reply:
Done in https://github.com/near/nearcore/pull/4115/commits/0da96728250c49c0dc6e6ae834d90cf9f43f343f